src/Discord.Net.Core/Entities/Applications/ApplicationInstallParams.cs

```diff
@@ -1,31 +1,26 @@
-using System;
 using System.Collections.Generic;
 using System.Collections.Immutable;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
-namespace Discord
+namespace Discord;
+
+/// <summary>
+///     Represents install parameters for an application.
+/// </summary>
+public class ApplicationInstallParams
 {
     /// <summary>
-    ///     Represents install parameters for an application.
+    ///     Gets the scopes to install this application.
     /// </summary>
-    public class ApplicationInstallParams
-    {
-        /// <summary>
-        ///     Gets the scopes to install this application.
-        /// </summary>
-        public IReadOnlyCollection<string> Scopes { get; }
+    public IReadOnlyCollection<string> Scopes { get; }
 
-        /// <summary>
-        ///     Gets the default permissions to install this application
-        /// </summary>
-        public GuildPermission? Permission { get; }
+    /// <summary>
+    ///     Gets the default permissions to install this application.
+    /// </summary>
+    public GuildPermission Permission { get; }
 
-        internal ApplicationInstallParams(string[] scopes, GuildPermission? permission)
-        {
-            Scopes = scopes.ToImmutableArray();
-            Permission = permission;
-        }
+    public ApplicationInstallParams(string[] scopes, GuildPermission permission)
```

Comment:
Might want to add a null check to `scopes`, each item in `scopes`, and maybe even validation of scopes (e.g `application.blow_up` isn't a valid scope)